### PR TITLE
 ducking implementated in jump.py

### PR DESCRIPTION
**Pull request by Phuc-Nguyenn**

### CODE REVIEW

**Comment by Phuc-Nguyenn** [comment]
added jumping mechanics in jump.py

File: src/main.py
Comment (rayokamoto) [inline src/main.py:1]: Please revert the change in this file

File: src/jump.py
Comment (rayokamoto) [inline src/jump.py:1]: Please have strings use double quotes, not single quotes.

**Review by rayokamoto — CHANGES_REQUESTED** [review]

**Review by rayokamoto — APPROVED** [review]
